tests/schema/fail/media-type-enc-item-exclusion.yaml

```
@@ -0,0 +1,10 @@
+openapi: 3.2.0
+info:
+  title: API
+  version: 1.0.0
+components:
+  requestBodies:
+    content:
+      multipart/mixed:
+        encoding: {}
+        itemEncoding: {}
```

```suggestion
    encoding-with-itemEncoding-not-allowed:
      content:
        multipart/mixed:
          encoding: {}
          itemEncoding: {}
```
Name of reusable request body is missing, making this fail for the wrong reason.